internal/ui/app.go

```diff
@@ -199,6 +201,23 @@ func (a *App) quitCmd(evt *tcell.EventKey) *tcell.EventKey {
 	return nil
 }
 
+func (a *App) suspendCmd(evt *tcell.EventKey) *tcell.EventKey {
+	if a.InCmdMode() {
+		return evt
+	}
+
+	if !a.Config.K9s.NoSuspendOnCtrlZ {
+		a.Suspend(func() {
```

Comment:
This will require additional testing on various platforms to make sure we're cool on other non nix aka windows?

Reply:
I've got Windows 10 as dual boot, so can give it a test. Is there are set of other recommended platforms/versions? 
Some VMs/containers might be required for this.